algorithms/backtrack/letter_combination.py

```diff
@@ -7,7 +7,7 @@
 """
 
 
-def letter_combinations(digits:"str")->"List[str]":
+def letter_combinations(digits: str)->list[str]:
```

Comment:
I think typing hint `List[str]` was correct. (don't know if `list[str]` is also possible). But for that we should also import `from typing import List`.

I suggest we annotate the types of args in the docstrings (the old way), mainly because `typing` is only supported by Python 3.5+, and also for consistency. :)

Reply:
i agree. will remove